### DISCUSSION TITLE
config file missing/unable to configure picom

### DISCUSSION THREAD

**Discussion by ANP-Oxy** [yshui/picom#1268] [Q&A]
I'm running PopOS 22.04. I'm using picom for configuring my suckless setup of dwm dmenu and st. 

I installed Picom using apt. Then when I looked for the config file in `/etc/xdg/`  there was no `picom.conf` file there. 

I downloaded the `picom.sample.conf` from the github repo and put it in `.config`.
as soon as I do that the picom just breaks. and throws this error 
```[ 05/26/2024 00:10:03.411 c2_parse_target ERROR ] Pattern "_GTK_FRAME_EXTENTS@" pos 19: Target type cannot be determined.```

Without config file picom works fine but if I try to put a config file it starts throwing this error. 

If I remove the config file and just do `picom -b` it starts working again. although it does show few messages like 
```
MESA: error: ZINK: failed to choose pdev
glx: failed to create drisw screen
```
Also im not sure if this is relevant but I patched dwm with a patch called clientopacity and changed opacity in dwm config but nothing happened. But when I installed picom the the opacity that I set in dwm config automatically started working. So i don't really know how that works.

**Comment by antnesswcm** [comment]
You should check more picom logs and adjust the level to info.  
picom.sample.conf should be renamed to picom.conf (I'm sure you've done this).  
If --config is not used, picom will look for a configuration file in XDG_CONFIG_HOME/picom.conf (~/.config/picom.conf, usually), then XDG_CONFIG_HOME/picom/picom.conf, then XDG_CONFIG_DIRS/picom.conf (often /etc/xdg/picom.conf), then XDG_CONFIG_DIRS/picom/picom.conf.  
Additionally, you can confirm your picom version.
I am using DWM, ST, etc. on Arch, and I am researching how to configure Picom. Good luck to me
